contracts/tasks/helpers/constants.ts

```diff
@@ -3,27 +3,17 @@
  */
 export enum ESupportedChains {
   Sepolia = "sepolia",
-  Goerli = "goerli",
-  Mainnet = "mainnet",
   Coverage = "coverage",
   Hardhat = "hardhat",
-  OptimismGoerli = "optimism_goerli",
-  OptimismSepolia = "optimism_sepolia",
-  Optimism = "optimism",
 }
 
 /**
  * Supported network chain ids for deployment and task running
  */
 export enum EChainId {
   Hardhat = 31337,
-  Goerli = 5,
   Sepolia = 11155111,
-  Mainnet = 1,
   Coverage = 1337,
-  OptimismGoerli = 420,
```

Comment:
These ones are not supported with current deploy workflow so I just removed them.

Reply:
wdym?

Reply:
So, for now it's possible to run hardhat tasks only for 3 networks: hardhat, coverage and sepolia.
Deployment is not supported for other networks yet so that's why I removed unsupported networks.
In the future, when we support OP testnets, these values will be restored.

Reply:
One thing why I dropped OP testnets - it's hard to get native tokes for deploy (like 1e-4 ETH per 24 hours)

Reply:
https://optimism-faucet.com/ can get up to 0.5ETH a day btw